apps/frontend/src/app/tournament-admin/tournament-overview/tournament-overview.component.html

```diff
@@ -30,6 +30,6 @@ <h2>Sharing:</h2>
     <p>Player Code: work in progress</p>
   </div>
   <div class="footer">
-    <button mat-raised-button style="margin: 1vh 40vh 0 0" type="submit" [disabled]="!this.myForm.valid" (click)="onSubmit()">Save</button>
+    <button mat-raised-button style="margin: 1vh 40vh 0 0" type="submit" [disabled]="!myForm.valid">Save</button>
```

Comment:
Since I just saw this here:
Please don't use heading elements (h2 in this case) as form labels. You should use the `label` element instead.

Here's a great resource on how to build the correct semantic HTML for forms: https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form